Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Events/InteractableOnClickEffect.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// A basic receiver for detecting clicks
+    /// </summary>
+    public class InteractableOnClickEffect : ReceiverBase
```

Comment:
It's not clear to me from the description + name what this does. I don't think the class only receives clicks, it seems to also perform some effect in response to the clicks. It seems like this is something that spawns a gameobject in response to a click. Is that something we want in the core SDK, or something that would make more sense as an example?

Reply:
This should be another PR and definitely failed on the description.

Reply:
removed